model/client.go

```diff
@@ -2334,3 +2334,25 @@ func (c *Client) ListReactions(channelId string, postId string) ([]*Reaction, *A
 		return ReactionsFromJson(r.Body), nil
 	}
 }
+
```

Comment:
Sorry just one more thing, can you add a comment detailing what the function does, returns, etc.?